Makefile

```diff
@@ -136,6 +136,11 @@ cache/bats-core-$(BATS_COMMIT).tar.gz:
 	# Use 2opremio's fork until https://github.com/bats-core/bats-core/pull/255 is merged
 	curl --fail -L -o $@ https://github.com/2opremio/bats-core/archive/$(BATS_COMMIT).tar.gz
 
+generate: generate-go generate-deploy
+
+generate-go:
```

Comment:
I made this same addition in #270 but called it `generate-codegen` there, which is a bit more descriptive to me personally.

Reply:
As usual naming stuff is hard :smile: updated!